internal/provider/cluster_resource.go

```diff
@@ -218,7 +219,13 @@ func (r *clusterResource) Schema(
 					"disk_iops": schema.Int64Attribute{
 						Optional:    true,
 						Computed:    true,
-						Description: "Number of disk I/O operations per second that are permitted on each node in the cluster. Zero indicates the cloud provider-specific default.",
+						Validators:  []validator.Int64{
+							// If supplied this value must be non-zero. 0 is a
+							// valid api value indicating the default being
+							// returned but it causes a provider inconsistency.
+							int64validator.AtLeast(1),
```

Comment:
Hmm this is a breaking change if anyone is actually setting this right? Is it really important to change this? It seems like anyone setting zero today knows what they're getting. I don't feel that strongly about it though so if you want to merge it then go for it.

Reply:
As discussed offline, I don't think this is a breaking change because using 0 as was previously mentioned in the doc causes an error.  When 0 is written to the tf configuration, it is then passed via the api.  The api returns the actual (non-zero) default depending on the disk size.  The 0 in the configuration causes a state inconsistency error with the returned value.  I just noticed the wrong commit message was added to this PR since I broke out some vendor additions to a separate commit.  I'll fix the PR message. Hopefully that helps clear this point up.